### DISCUSSION TITLE
block direct access to stream url but allow embedding from whitelisted domains

### DISCUSSION THREAD

**Discussion by StephanHi** [owncast/owncast#1780] [Q&A] [answered]
Hi, I would like to prevent direct access to the stream url but allow embedding the url in a whitelisted domain (another website where I deal with user management and payment stuff). Much like vimeo does (where I can restrict the access to videos and livestreams to whitelisted domains). How can this be achieved?

**Comment by gabek** [comment, marked as answer]
You'd want to to do this via an HTTP proxy. Owncast doesn't itself have any kind of whitelisting or restrictions.

**Reply by StephanHi** [reply]
Thank you very much, I will follow this approach.
